Source/RunActivity/Viewer3D/Cameras.cs

```diff
@@ -1080,7 +1080,7 @@ protected override void OnActivate(bool sameCamera)
             }
             else
             {
-                isVisibleTrainCarViewerOrWebpage = (Viewer.TrainCarOperationsWindow.Visible && !Viewer.TrainCarOperationsViewerWindow.Visible) || Viewer.TrainCarOperationsViewerWindow.Visible || (Viewer.TrainCarOperationsWebpage.Connections > 0 && Viewer.TrainCarOperationsWebpage.TrainCarSelected);
+                isVisibleTrainCarViewerOrWebpage = (Viewer.TrainCarOperationsWindow.Visible && !Viewer.TrainCarOperationsViewerWindow.Visible) || Viewer.TrainCarOperationsViewerWindow.Visible || (Viewer.TrainCarOperationsWebpage?.Connections > 0 && Viewer.TrainCarOperationsWebpage.TrainCarSelected);
```

Comment:
Looks like this question mark is not that necessary. That's because the else belongs to the if condition "Viewer.TrainCarOperationsWebpage == null". So in the else the varioable TrainCarOperationsWebpage  is not null. It does not harm either.